app/models/station.rb

```diff
@@ -5,6 +5,6 @@ class Station < ApplicationRecord
                         :dock_count,
                         :city,
                         :installation_date
-  has_many :station_trips
-  has_many :trips, through: :station_trips
+  has_many :start_trips, class_name: 'Trip', foreign_key: :start_station_id
```

Comment:
When I was using this class_name syntax, I was getting a depreciation warning ... wonder if it's still needed here as cleaning my migration up fixed this.